scripts/capture_pane_helpers.sh

```diff
@@ -4,7 +4,10 @@
 # - name_option & default_name
 
 get_path() {
-	get_tmux_option "$path_option" "$default_path"
+	local path_template=$(get_tmux_option "$path_option" "$default_path")
+	local check_folder=$(tmux display-message -p "$path_template")
```

Comment:
What does this line do? Why can't you just use `path_template` variable on the 2 lines below?

Reply:
The reason of this is that path_template contains "%Y-%m-%d" which we need to convert real values. It can be done only through tmux display-message because bash doesn't know about this anything